elasticsearch/templates/participants_structured.json

```diff
@@ -344,6 +344,30 @@
                     }
                   }
                 },
+                "LONGITUDINAL_VACCINE_RECEIVED_LIST": {
```

Comment:
Explicitly call out these composite questions in mapping to export as text in new style